conf/application.conf

```diff
@@ -40,7 +40,7 @@ akka {
 # ~~~~~
 # The secret key is used to sign Play's session cookie.
 # This must be changed for production, but we don't recommend you change it in this file.
-play.http.secret.key = "changeme"
+play.http.secret.key = "13245749837902457483624895746385723648"
```

Comment:
kan je deze waarde niet in production.conf zetten? Dan kunnen 'secret' keys niet in de github repo komen

Reply:
Ja, dit hoort hier sowieso niet thuis.